rrfs-test/ush/mpasjedi_increment_singleob.py

```diff
@@ -39,11 +39,12 @@
 
 # JEDI data
 datapath = "./"
-janalysis   = "./an.2022-05-26_19.00.00.nc"            # analysis file
-jbackgrnd   = "./Data/bkg/bg.2022-05-26_19.00.00.nc"            # background file
-jstatic      = "./Data_static/static.93175.nc"                         # to load the MPAS lat/lon
```

Comment:
Thanks, @SamuelDegelia-NOAA What change we should make? Do you mean we need to make this file also work for the previous MPASJEDI case? 

I have suggested I keep a standalone copy of this file for the new MPASJEDI case. But after discussion with Donnie, we went another route.  I think it is still a good idea  that we don't change the behavior of the previous MPASJEDI case. I will make a change and keep two copies of this file. Does this work for you?

Reply:
Sorry @guoqing-noaa, I think my review comments just showed up despite typing them a couple of weeks ago because I still had my review as "pending". I am still learning how to use Github, hah. 

But anyways I just meant to also edit `mpasjedi_increment_fulldom.py` to be consistent with these changes to  `mpasjedi_increment_singleob.py` for the new MPAS-JEDI case. This new case changes the analysis filename prefix from `an.${DATE}` to `ana.${DATE}` and changes the path for these static files. This change to the singleob plotting script highlighted here represents those new file names, but `mpasjedi_increment_fulldom.py` still uses the original names. So basically I am just asking to make these same changes to `mpasjedi_increment_fulldom.py`.

Reply:
Ah, I got it. Thanks for the clarification and mentioning this. I forgot that part.